src/NameEntry.tsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+
+const NameEntry: React.FunctionComponent = () => {
+  const [id, setId] = React.useState("");
+  const [name, setName] = React.useState("");
+  const edit: (
+    setter: React.Dispatch<React.SetStateAction<string>>
```

Comment:
declares.d.ts にエイリアスがあるのでこれを使ってください。
```suggestion
    setter: SetStateFunc<string>
```